strongbox-storage/strongbox-storage-api/src/test/java/org/carlspring/strongbox/configuration/ConfigurationManagerTest.java

```diff
@@ -66,21 +62,32 @@
 
     private YAMLMapper yamlMapper;
 
+    private Path outputFilePath;
 
     @BeforeEach
-    public void setUp()
+    void setUp(TestInfo testInfo)
             throws IOException
     {
-        Path yamlPath = Paths.get(CONFIGURATION_BASEDIR);
-        if (Files.notExists(yamlPath))
+        Path configurationBasePath = getConfigurationBasePath();
+        if (Files.notExists(configurationBasePath))
         {
-            Files.createDirectories(yamlPath);
+            Files.createDirectories(configurationBasePath);
         }
 
+        outputFilePath = getConfigurationOutputFilePath(testInfo);
+
         yamlMapper = yamlMapperFactory.create(
                 Sets.newHashSet(CustomRepositoryConfigurationDto.class, RemoteRepositoryConfigurationDto.class));
     }
 
+    @AfterEach
+    void tearDown()
```

Comment:
@ptirador we should use `TestInfo` here as well

Reply:
when we have parallel execution, one test can delete configuration file of another test